q2_fmt/plugin_setup.py

```diff
@@ -342,28 +310,28 @@
         'sampling_depth': sampling_depth
     },
     output_descriptions={
-        'peds_dists': peds_dists
+        'prdf_dists': pedf_dists
```

Comment:
Should this be `prdf_dists` instead of `pedf_dists`?

Reply:
this is a hard question i guess, 
This dist are literally the same and ive always used the same "string" named peds_dist as the output description. I'll change it but I am not sure how yet